### ISSUE TITLE
`can't access property "length", a.sharedService.visibleColumns is undefined` when hiding column

### ISSUE THREAD

**Issue by Elias-Graf** [ghiscoding/Angular-Slickgrid#1180]
### Describe the bug

The error `can't access property "length", a.sharedService.visibleColumns is undefined` is printed to the console, after moving a column, and then hiding one of the columns.

### Reproduction

1. Go to: https://ghiscoding.github.io/Angular-Slickgrid/#/draggrouping
2. Move "Title" column to the second position (after: "Duration")
3. Hide the "Title" column (or any other) via right-click and removing the check

### Expectation

No error should be thrown.

### Environment Info

```shell
Environment: https://ghiscoding.github.io/Angular-Slickgrid/#/draggrouping
Browser: Firefox / Chrome
OS: Windows 11
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/Angular-Slickgrid/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [HOWTO - Step by Step](https://github.com/ghiscoding/angular-slickgrid/wiki/HOWTO---Step-by-Step).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/Angular-Slickgrid/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/Angular-Slickgrid/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
fixed in v6.0.1

**Comment by ghiscoding** [comment]
you could also star the project if you like the lib 😉 ⭐ 

Cheers
